detekt-rules/src/test/kotlin/OrtPackageNamingTest.kt

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.detekt
+
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.test.lint
+
+import io.kotest.core.TestConfiguration
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.collections.beEmpty
+import io.kotest.matchers.collections.haveSize
+import io.kotest.matchers.should
+
+import java.io.File
+
+import org.ossreviewtoolkit.utils.common.safeDeleteRecursively
+import org.ossreviewtoolkit.utils.common.safeMkdirs
+
+class OrtPackageNamingTest : WordSpec({
+    val rule = OrtPackageNaming(Config.empty)
+
+    "OrtPackageNaming rule" should {
+        "succeed for package in top-level module" {
+            val file = createFile(
+                dir = "model/src/main/kotlin",
+                content = """
+                    package org.ossreviewtoolkit.model
+                """.trimIndent()
```

Comment:
`content` is never more than a single line in any of the tests. Would it thus make sense to not use a multi-line string for it for more compact code?

Reply:
Done in #6573.